docs/development.md

```diff
@@ -168,3 +170,75 @@ In the vagrant box, find the Hammer repositories at `/home/vagrant/` and the
 configuration at `/home/vagrant/.hammer`. Specifically, to change the Foreman
 instance Hammer points to, update
 `/home/vagrant/.hammer/cli.modules.d/foreman.yml`.
+
+## Capsule Development
+
+To use this functionality, add the following configuration to your boxes.yaml,
+changing the hostnames as needed
+
+### To setup a capsule with an existing development environment
+
+* Add to existing development server's configuration in boxes.yaml
+```
+  ansible:
+    group: 'server'
+```
+* Add capsule with development server's name in server to boxes.yaml:
+
+```
+foocapsule:
+  box: centos7
+  ansible:
+    playbook: 'playbooks/capsule-dev.yml'
+    group: 'capsule'
+    server: 'foo'
+```
+* ssh into existing development server and ```rails s```
+* spin up new capsule ```vagrant up foocapsule```
+
+### To setup a capsule and a new development environment
+
+* setup boxes.yaml
+
+```
+foo:
```

Comment:
I'd align this with ```boxes.yaml.example```. That appears to be missing the ansible option so maybe add that there as well?